### ISSUE TITLE
plain/text mail quotation does not match HTML mail quotation

### ISSUE THREAD

**Issue by Normo** [zammad/zammad#3975]
### Infos:

* Used Zammad version: 5.0.3-1639085449.b757f8e2.bionic
* Installation method (source, package, ..): package
* Operating system: Ubuntu 18.04.6 LTS
* Database + version: PostgreSQL 10.19
* Elasticsearch version: 7.15.2
* Browser + version: Firefox 96.0
* Mail client: Thunderbird 91.5.0

### Expected behavior:

* quotation in plain text mail matches quotation in HTML mail

### Actual behavior:

* quotation in plain text mail is broken and does match quotation in HTML mail
<img src="https://user-images.githubusercontent.com/2527899/155302526-6119df18-0696-4496-8589-ced7089a2b43.png" alt="Plain text mail" width=400><img src="https://user-images.githubusercontent.com/2527899/155302547-714532bc-6f1b-4b03-9e37-6ce9be3f9e27.png" alt="HTML mail" width=400>

### Steps to reproduce the behavior:

* send reply to a ticket
* view mail message body as plain/text
* view mail message body as original html and compare 

Yes I'm sure this is a bug and no feature request or a general question.

**Comment by MrGeneration** [comment]
What software are you looking at these notification with?

**Comment by Normo** [comment]
@MrGeneration 
Sorry, I totally forgot to mention: Thunderbird 91.5.0 (64-bit)

**Comment by Normo** [comment]
@MrGeneration Could you please remove the *waiting for feedback* label?

**Comment by MrGeneration** [comment]
Bumping issues doesn't help, but sure.

**Comment by cautopartes** [comment]
### Tldr.: Verified
### Long version: 
The "Steps to reproduce the behavior" leave out some details, so here is what I did to reproduce the issue: 
0. We are talking about agent notifications. At least that is what I see in the screenshots, thus what I tested. 

## First run
1. Created ticket in Zammad (6.2.0, hosted/SaaS)
2. Sent reply from Thunderbird (115.8.0 (64-bit)) in plain text format.
-> Agent notification in Thunderbird, as HTML:
![image](https://github.com/zammad/zammad/assets/69846837/29b19bcf-7523-42ad-8940-5142e38f6694)
-> Agent notification in Thunderbird, as plain text:
![image](https://github.com/zammad/zammad/assets/69846837/e76c98dd-fba8-408f-885c-462011cbfa1e)
 
## Second run
1. Created ticket in Zammad (6.2.0, hosted/SaaS)
2. Sent reply from Thunderbird in HTML format.
-> Agent notification in Thunderbird, as HTML:
![image](https://github.com/zammad/zammad/assets/69846837/f0110f7f-79c0-4825-a8a3-550f8e517e60)
-> Agent notification in Thunderbird, as plain text:
![image](https://github.com/zammad/zammad/assets/69846837/2b488e35-7006-4226-a010-d19d68b2f02f)

## Conclusion:
Agent notifications for communication via HTML emails viewed in Thunderbird as plain text, do not represent the quoting correctly.
